pdc/apps/release/tests.py

```diff
@@ -1196,7 +1207,7 @@ def test_update_can_explicitly_erase_optional_field(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertIsNone(response.data.get('dist_git'))
         self.assertIsNone(models.Release.objects.get(release_id='release-1.0').dist_git_branch)
-        self.assertNumChanges([1])
+        self.assertNumChanges([2])
```

Comment:
What's the additional change about?

Reply:
I was also confused here. After I print the changeset, I really saw two changes.
Here it is:
[{'old_value': u'{"release_id": "release-1.0", "product_version": null, "integrated_with": null, "sigkey": null, "active": true, "allow_buildroot_push": false, "short": "release", "name": "Test Release", "allowed_debuginfo_services": [], "version": "1.0", "allowed_push_targets": [], "base_product": null, "release_type": "ga"}', 'target_id': 1, 'new_value': u'{"release_id": "release-1.0", "product_version": null, "integrated_with": null, "sigkey": "ABCDEF", "active": true, "allow_buildroot_push": false, "short": "release", "name": "Test Release", "allowed_debuginfo_services": [], "version": "1.0", "allowed_push_targets": [], "base_product": null, "release_type": "ga"}', 'target_class': u'release', u'changeset_id': 1, u'id': 1}, 
{'old_value': u'{"dist_git_branch": "release_branch", "release_id": "release-1.0"}', 'target_id': 1, 'new_value': u'null', 'target_class': u'releasedistgitmapping', u'changeset_id': 1, u'id': 2}]

I think the original one is just dist_git_branch change, but after set the default sigkey, here is one release change.

Reply:
This is correct: dist git mapping is stored in a separate model, and now also the release itself is changing.